app/src/main/res/values/strings.xml

```diff
@@ -45,6 +45,9 @@
     <string name="expected_ovulation_color">Expected Ovulation</string>
     <string name="reminders">Reminders</string>
     <string name="days_before_reminder">Days Before Reminder</string>
+    <string name="period_notification_message">Period Notification Message</string>
```

Comment:
I would suggest having "Your period is about to start" as this is the text today.